### ISSUE TITLE
NTR: C-terminal of phosphate-regulating hormone high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8407]
New term request C-terminal of phosphate-regulating hormone high in blood (elthat_180527094611): Fibroblast growth factor 23; FGF23  UMLS: 1968909

**Comment by pnrobinson** [comment]
@MickeySegal
can you clarify what this is? Are we referring to https://pubmed.ncbi.nlm.nih.gov/16869716/

**Comment by MickeySegal** [comment]
We use this in https://www.ncbi.nlm.nih.gov/books/NBK476672/ "Elevated plasma levels of the C-terminal portion of the phosphate-regulating hormone, fibroblast growth factor 23 (FGF23): measured (in clinical laboratories) by enzyme-linked immunosorbent assay (ELISA) utilizing antibodies that bind to the C-terminal portion of FGF23; detecting the combination of the biologically active intact hormone and the biologically inactive C-terminal fragments"

BTW, our finding is now "Fibroblast growth factor 23 hormone C-terminal fragments high in blood"

**Comment by pnrobinson** [comment]
@MickeySegal 
not an expert, but here it is said that the C-terminal fragments are biologically inactive:
Moreover, to determine whether the choice of FGF‐23 assay affects interpretation of its physiological role, we measured FGF‐23 using one assay that detects both biologically active intact FGF‐23 peptide and carboxyl terminal fragments that are presumed to be biologically inactive, and a second assay that detects only intact FGF‐23 peptide.
https://pubmed.ncbi.nlm.nih.gov/16869716/

Is this the test that you are looking for? Would it be better to have

"Elevated circulating FGF‐23 peptide concentration"
def: The concentration of fibroblast growth factor 23 (FGF‐23) in the blood circulation is above the upper limit of normal. Comment: FGF23 is a regulator of phosphate metabolism. elevated FGF‐23 levels may induce phosphate wasting.
PMID:16869716

**Comment by MickeySegal** [comment]
Both intact and C-terminal are tested; intact is low and C-terminal is high:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3233725/
"HFTC patients with loss of function mutations in GALNT3 have a remarkably similar serum FGF23 profile to those with FGF23 mutations, manifesting low serum intact FGF23, but significantly elevated C-terminal FGF23 concentrations [[37](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3233725/#R37)]. In vitro, deleting GALNT3 causes increased proteolytic cleavage and the appearance of FGF23 immunoreactive polypeptide fragments with reduced intact FGF23 secretion [[38](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3233725/#R38)]. The biochemical profile of elevated plasma C-terminal FGF23 in conjunction with low intact FGF23 in the setting of hyperphosphatemia appears to be virtually diagnostic for GALNT3- and FGF23-TC (hFTC due to αKL mutations is described below)."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000755
- label: Elevated circulating C-terminal fibroblast growth factor 23 concentration
- parents: [Abnormal circulating fibroblast growth factor 23 concentration (HP:6000489)](https://hpo.jax.org/app/browse/term/HP:6000489)
- pmid: PMID:16869716
- def: The concentration of the C-terminal portion of fibroblast growth factor 23 in the blood circulation is outside of the limits of normal.
- comment: FGF23 is also known as phosphate-regulating hormone.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8407
- synonyms:
